activities/Markdown.activity/js/activity.js

```diff
@@ -1,4 +1,4 @@
-define(["sugar-web/activity/activity","sugar-web/datastore"], function (activity, datastore) {
+define(["sugar-web/activity/activity","webL10n","sugar-web/datastore"], function (activity, datastore) {
```

Comment:
You should add a l10n parameter between activity and datastore else you break the passing of parameters